models/benchmarks/benchmark_model.py

```diff
@@ -39,5 +41,18 @@ def get_plugin(self):
         """Returns the plugin to parse the results"""
         pass
 
+    def parse_run_args(self, run_args):
+        parser = argparse.ArgumentParser()
+        parser.add_argument('--iterations', '-i', type=str, default='1',
+                            help='Number of times to run each instance')
+        parser.add_argument('--size', '-s', type=str, default='0',
```

Comment:
Some benchmarks have sizes per dimension, others have a unique value to control them all. This really isn't going to work, no matter how much we try. We can think about how this fits in the metadata later, right now, we just need iterations.

Same comment for complexity.